nni/algorithms/compression/pytorch/pruning/dependency_aware_pruner.py

```diff
@@ -27,7 +27,7 @@ class DependencyAwarePruner(Pruner):
     """
 
     def __init__(self, model, config_list, optimizer=None, pruning_algorithm='level', dependency_aware=False,
-                 dummy_input=None, **algo_kwargs):
+                 dummy_input=None, global_sort=False, **algo_kwargs):
```

Comment:
Do all the dependency pruners support `global_sort` mode? If not, I'm a little concerned.

Reply:
agree, how many pruners support `global_sort`?

Reply:
Not all of them support `global_sort`, have deleted parameters in them.